apps/etterlatte-beregning/src/main/kotlin/beregning/grunnlag/BeregningsGrunnlagRoutes.kt

```diff
@@ -121,6 +122,16 @@ fun Route.beregningsGrunnlag(
                 call.respond(HttpStatusCode.OK, grunnlag)
             }
         }
+
+        put("/{$BEHANDLINGID_CALL_PARAMETER}/overstyr/reguler") {
+            withBehandlingId(behandlingKlient, skrivetilgang = true) { behandlingId ->
+                logger.info("Regulerer overstyr grunnlag for behandling $behandlingId")
```

Comment:
Denne fikser bare grunnlaget -- og vi må fremdeles beregne med nytt grunnlag? Jeg tror det er lurt å kommunisere det ganske eksplisitt

Reply:
Jeg var litt usikker på hva som var riktig av post/put/patch. Med tanke på at å kalle endepunktet vil gi samme resultat ved flere kall så tenkte jeg at put var riktig. Men mulig patch er bedre.

Og ja det beregner ikke behandlingen men dette endepunktet brukes av reguleringsflyten og beregning er et eget steg/endepunkt-kall.
Men kan kommentere eller gjøre noe annet for å tydeliggjøre det.

Reply:
Metoden virker rimelig :D Var mest kontrasten mellom "put" + `reguler` i route, vs. loggmeldingen som snakker om grunnlaget som gjorde at jeg ble litt usikker